docs/manual/tutorials/admin/advanced-config/airtime-api-authentication/index.md

```diff
@@ -0,0 +1,40 @@
+# Tutorials
```

Comment:
Why have you copied the tutorials here?

Reply:
The goal was to organize the folder tree so that all tutorials would be located within the tutorials folder